user.go

```diff
@@ -519,3 +519,46 @@ func (cli *Client) usync(ctx context.Context, jids []types.JID, mode, context st
 		return &list, err
 	}
 }
+
+func (cli *Client) UnblockUser(target types.JID) (Blockedlist []types.JID, err error) {
+	return cli.updateBlock(target, false)
+}
+
+func (cli *Client) BlockUser(target types.JID) (Blockedlist []types.JID, err error) {
+	return cli.updateBlock(target, true)
+}
```

Comment:
```suggestion
func (cli *Client) UnblockUser(target types.JID) ([]types.JID, error) {
	return cli.updateBlock(target, false)
}

func (cli *Client) BlockUser(target types.JID) ([]types.JID, error) {
	return cli.updateBlock(target, true)
}
```

That return variable name is wrong (shouldn't start with uppercase) and it's unnecessary anyway